### PR TITLE
 Portuguese Translation and new method for sequences 

### PR DESCRIPTION
**Pull request by jjnf**

### CODE REVIEW

File: hotel_restaurant/models/hotel_restaurant.py [hunk 1]
```diff
@@ -161,7 +161,8 @@ def table_done(self):
     _name = "hotel.restaurant.reservation"
     _description = "Includes Hotel Restaurant Reservation"
 
-    reservation_id = fields.Char('Reservation No', size=64, required=True, default=lambda obj: obj.env['ir.sequence'].get('hotel.restaurant.reservation'))
+    #reservation_id = fields.Char('Reservation No', size=64, required=True, default=lambda obj: obj.env['ir.sequence'].get('hotel.restaurant.reservation'))
+    reservation_id = fields.Char('Reservation No', size=64, required=True)
     room_no = fields.Many2one('hotel.room', string='Room No', size=64)
```
Comment (JayVora-SerpentCS) [inline hotel_restaurant/models/hotel_restaurant.py]: Why do we need to remove the auto seq?

File: hotel_restaurant/models/hotel_restaurant.py [hunk 2]
```diff
@@ -170,6 +171,14 @@ def table_done(self):
...
     state = fields.Selection([('draft', 'Draft'), ('confirm', 'Confirmed'), ('done', 'Done'), ('cancel', 'Cancelled')], 'state', select=True, required=True, readonly=True,default=lambda * a: 'draft')
 
+    #new method for sequence
+    def create(self, cr, uid, vals, context=None):
+        if not vals:
+            vals = {}
+        if context is None:
+            context = {}
+        vals['reservation_id'] = self.pool.get('ir.sequence').get(cr, uid, 'hotel.restaurant.reservation')
+        return super(hotel_restaurant_reservation, self).create(cr, uid, vals, context=context)
```
Comment (JayVora-SerpentCS) [inline hotel_restaurant/models/hotel_restaurant.py]: We should not work on create(), rather to save the gap of sequence we should better handle the config on ir.sequence.

File: hotel_restaurant/models/hotel_restaurant.py [hunk 3]
```diff
@@ -263,6 +273,14 @@ def generate_kot(self):
...
     amount_total = fields.Float(compute=_total, method=True, string='Total')
 
+    #new method for sequence
+    def create(self, cr, uid, vals, context=None):
+        if not vals:
+            vals = {}
+        if context is None:
+            context = {}
+        vals['order_no'] = self.pool.get('ir.sequence').get(cr, uid, 'hotel.restaurant.order')
+        return super(hotel_restaurant_order, self).create(cr, uid, vals, context=context)
```
Comment (JayVora-SerpentCS) [inline hotel_restaurant/models/hotel_restaurant.py]: Try the @api v8 level plz

File: hotel_restaurant/models/hotel_restaurant.py (hunk 1 above)
Comment (jjnf) [inline hotel_restaurant/models/hotel_restaurant.py]: A numerical sequence for documents should create the number on the recording act in the database.
It's not what is happening. If I create a reserve is assigned a number and when I record changes the number.
If I discard the creation of reserves, the numerical sequence always advances a number.
In standar method this problem does not occur

**Pull request closed without merging**